spring-boot-autoconfigure/src/main/java/org/axonframework/boot/autoconfig/JdbcAutoConfiguration.java

```diff
@@ -16,12 +27,30 @@
 
 @ConditionalOnBean(DataSource.class)
 @Configuration
+@AutoConfigureAfter(JpaAutoConfiguration.class)
 public class JdbcAutoConfiguration {
 
+    @ConditionalOnMissingBean
```

Comment:
I think we should also add `EventStore` to this condition. If an Event Store is provided (e.g. AxonDB), we don’t need to configure a storage engine. 
(also check JPA config)